src/style.scss

```diff
@@ -75,28 +94,14 @@
   }
 }
 
+// Don't prevent overscroll on desktop devices
```

Comment:
The opposite actually, I think... (still trying to understand [it](https://developer.mozilla.org/en-US/docs/Web/CSS/overscroll-behavior))

Reply:
Right, I changed that